frontend/src/component/admin/instance-admin/InstanceStats/InstanceStats.tsx

```diff
@@ -0,0 +1,82 @@
+import { Download } from '@mui/icons-material';
+import { Button, IconButton, Link, Table, TableBody, TableCell, TableHead, TableRow } from '@mui/material';
+import { Box } from '@mui/system';
+import { VFC } from 'react';
+import { useInstanceStats } from '../../../../hooks/api/getters/useInstanceStats/useInstanceStats';
+import { formatApiPath } from '../../../../utils/formatPath';
+import { PageContent } from '../../../common/PageContent/PageContent';
+import { PageHeader } from '../../../common/PageHeader/PageHeader';
+
+export const InstanceStats: VFC = () => {
+    const { stats, loading } = useInstanceStats();
+
+
+    let versionTitle;
+    let version;
+
+    if(stats?.versionEnterprise) {
+        versionTitle = 'Unleash Enterprise version';
+        version = stats.versionEnterprise;
+    } else {
+        versionTitle = 'Unleash OSS version';
+        version = stats?.versionOSS;
+    }
+
+    
+
+    const rows = [
+        {title: 'Instance Id', value: stats?.instanceId},
+        {title: versionTitle, value: version},
+        {title: 'Users', value: stats?.users},
+        {title: 'Feature toggles', value: stats?.featureToggles},
+        {title: 'Projects', value: stats?.projects},
+        {title: 'Environments', value: stats?.environments},
+        {title: 'Roles', value: stats?.roles},
+        {title: 'Groups', value: stats?.groups},
+        {title: 'Context fields', value: stats?.contextFields},
+        {title: 'Strategies', value: stats?.strategies},
+    ];
```

Comment:
Hmm. Prettier seems to not be working correctly here.

Reply:
I have not gotten my IDE to work correctly with prettier after the merge. This is how it looks after auto-format of this file.